docs/querying.md

```diff
@@ -135,6 +135,8 @@ $like: '%hat',         // LIKE '%hat'
 $notLike: '%hat'       // NOT LIKE '%hat'
 $iLike: '%hat'         // ILIKE '%hat' (case insensitive) (PG only)
 $notILike: '%hat'      // NOT ILIKE '%hat'  (PG only)
+$regexp: '^[h|a|t]'    // REGEXP '^[h|a|t]' (MySQL only)
+$notRegexp: '^[h|a|t]' // NOT REGEXP '^[h|a|t]' (MySQL only)
```

Comment:
Just thought about this more, it probably won't work if JavaScript regexp syntax is different than the SQL regexp syntax. But worth looking into